### ISSUE TITLE
Launching R from OSGeo4W shell

### ISSUE THREAD

**Issue by florisvdh** [rsbivand/rgrass#92]
Since OSGeo4W version 2, we are missing the former feature of being able to launch R from the OSGeo4W shell by just typing `R`. This was a great feature to users of the R package **rgrass**.

It is due to the fact that the `rbatch` directory is missing in `OSGeo4W/apps/`; it was [present in version 1](https://download.osgeo.org/osgeo4w/v1/x86_64/release/rbatch/) but [not in version 2](https://download.osgeo.org/osgeo4w/v2/x86_64/release/).

@jef-n could you bundle `rbatch` in version 2 please? This would bring GRASS in OSGeo4W on par with standalone GRASS. The latter has always bundled `rbatch`, allowing to launch R without needing the filepath to the program.

The source code is at https://github.com/OSGeo/grass/tree/main/mswindows/external/rbatch. Also pinging @hellik.

**Comment by jef-n** [comment]
The files are in apps/grass/grass*/bin/

**Comment by florisvdh** [comment]
Yep; indeed this means that R can be launched from _within_ a GRASS OSGeo4W session.

However we'd like to also have R available standalone inside OSGeo4W shell, as it worked [in OSGeo4W version 1](https://download.osgeo.org/osgeo4w/v1/x86_64/release/rbatch/), i.e. typing `R` in the OSGeo4W shell.

This is a second important use case we know from our winGRASS users.

It would allow to access GRASS GIS (and other OSGeo4W tools) from R only when you need it. The environment variables for e.g. GDAL, PROJ etc. are then set for use out of the box in R.

How should we proceed?
